BH/Modules/Item/Item.cpp

```diff
@@ -42,6 +45,29 @@ void Item::LoadConfig() {
 	Toggles["Item Close Notifications"] = BH::config->ReadToggle("Item Close Notifications", "None", false);
 	Toggles["Allow Unknown Items"] = BH::config->ReadToggle("Allow Unknown Items", "None", false);
 
+	// To enable good character skills and good skill tab lists
+	Toggles["Skills"] = BH::config->ReadToggle("Skills", "None", false);

```

Comment:
Are these toggles needed? Will it work (without crashing) when `SkillsList` and `TabSkillsList` do not exist in the cfg?

Reply:
They are not needed. It does not crash when they are not there.
The GOODSK and GOODTBSK just won't work, exactly as if you have them set to false.

Reply:
BTW, the convention for reading config values changed with McGod's PR. This won't work as is now.